end-to-end.test.js

```diff
@@ -1,10 +1,27 @@
+async function getMetaTagContent(page, property) {
+  return await page.$eval(
+    `meta[property='${property}']`,
+    (element) => element.content
+  );
+}
+
 describe("End to End", () => {
   test("Home Page", async () => {
     await page.goto("http://localhost:5000/");
     await expect(page).toMatch("Contentz Test");
     await expect(page).toMatch(
       "Create Content, Get a Highly Optimized Website"
     );
+    const ogURLContent = await getMetaTagContent(page, "og:url");
+    expect(ogURLContent).toMatch("https://contentz.tech/home/");
```

Comment:
The OG url is wrong I think? It should be / not /home, right?

Reply:
😮 That was how it was working before, I'm going to change it